testutils/src/main/java/com/hyperwallet/android/ui/testutils/util/DateConversion.java

```diff
@@ -0,0 +1,22 @@
+package com.hyperwallet.android.ui.testutils.util;
+
+
+import static android.text.format.DateUtils.formatDateTime;
+
+import android.content.Context;
+
+import com.hyperwallet.android.ui.common.util.DateUtils;
+
+import java.util.Date;
+
+public class DateConversion {
+
+    public static String convertDate(Context context, String dateString, int flag) {
```

Comment:
We can move this file into `hyperwallet-android-ui-sdk/receiptui/src/androidTest/java/com/hyperwallet/android/ui/receipt`.

Reply:
ok